src/js/profile/subcategory.js

```diff
@@ -92,20 +93,12 @@ export class Subcategory extends Component {
                     } else if (indicator.content_type == ContentBlock.BLOCK_TYPES.HTMLBlock) {
                         block = this.addHTMLBlock(indicatorContainer, indicator, title, isLast);
                     }
-                    
+
                     this._indicators.push(block);
 
-                    
-                    
                     index++;
-                } else {
-                    $(wrapper).find(descriptionTextClass).text('No data available for this indicator for this geographic area');
-                    $(wrapper).find(descriptionClass).removeClass('hidden')
                 }
             }
-        } else {
-            $(wrapper).find(descriptionTextClass).text('No data available for this indicator for this geographic area');
-            $(wrapper).find(descriptionClass).removeClass('hidden')
```

Comment:
Why do we display an indicator if it doesn't have data. Maybe it is better to keep the message or not display the empty indicator?

Reply:
I double-checked, this change is correct. The mistake was that these are being displayed outside the indicators for categories and not indicators